### ISSUE TITLE
Proposing an addition to Chapter 5 - "Outcome Reward Models"

### ISSUE THREAD

**Issue by yassersouri** [natolambert/rlhf-book#388]
I think it would be a good idea to highlight how at inference time the per-token logits from the reward head are aggregated to arrive at a final decision.
When I read that section it was not clear.

**Comment by natolambert** [comment]
@yassersouri can you say more and ideally with excerpts from codebases or papers? I think I see what you mean but could use the evidence

**Comment by yassersouri** [comment]
@natolambert I actually do not know myself. I read the OAI paper "Training verifiers to solve math word problems" but also it wasn't clear there.

One can do average, maximum or some other kind of aggression from the per-token predictions, or you can just take the task token's prediction.

**Comment by natolambert** [comment]
Yeah that's my read, for now I'll close this then unless someone can find more evidence.

**Comment by yassersouri** [comment]
Actually this is covered in the book! I had not read it yet.

Chapter 5 has a section: "Inference Across Reward Model Types" which covers outcome reward model inference.
